bazel

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+TMPDIR="${TMPDIR:-/tmp}"
+
+if [ -z "$BAZEL_BIN_LOC" ]; then
+  BAZEL_BIN_LOC=~/.bazel_binaries
+fi
+
+mkdir -p "$BAZEL_BIN_LOC"
+
+if [ "$(uname -s)" == "Linux" ]; then
+  export BAZEL_LAUNCHER_PLATFORM_NAME='linux'
+elif [ "$(uname -s)" == "Darwin" ]; then
+  export BAZEL_LAUNCHER_PLATFORM_NAME='darwin'
+else
```

Comment:
No support for windows huh?

Reply:
fair... umm... i've never actually used bazel on windows outside of wsl2